### DISCUSSION TITLE
Установка на личный сервер VPS, вопрос по инструкций.

### DISCUSSION THREAD

**Discussion by Murakumo-JP** [xtrime-ru/antizapret-vpn-docker#24] [Q&A] [answered]
Здраствуйте планирую арендовать сервер но в оригинальной инструкций у антизапрета пишут в требованьях к серверу такое 

> Личный сервер или VPS с виртуализацией XEN или KVM

Так как в инструкций у вас я не нашёл нечего подобного кроме установки докера, то я могу арендовать любой сервер VPS или всё ещё нужны те требованья ?
Раньше с таким не имел дел по этому решил уточнить у знающих людей.

**Comment by xtrime-ru** [comment, marked as answer]
Да, требования точно такие же. Но под них подходят все VPS от популярных хостеров. KVM означает, что есть полный контроль над системой. Можно ставить пакеты, обновлять саму систему и тд.

**Reply by Murakumo-JP** [reply]
Ох.. спасибо, значит буду сидеть разбираться.

**Reply by xtrime-ru** [reply]
Все что нужно:

1. Выбрать ubuntu24.04 (предпочтительно) или ubuntu22.04 в конфигураторе любого хостинг провайдера
2. Создать VPS
3. Скопировать адрес и пароль для входа
4. Зайти по ssh на сервер
5. выполнить эти 3 шага (6 комманд): https://github.com/xtrime-ru/antizapret-vpn-docker?tab=readme-ov-file#installation

**Reply by Murakumo-JP** [reply]
Большое вам спасибо, выбрал специально ваш контейнер потому что он по свежей как я понял по форуму ntc.party 🙏
